native-schema-registry/python/PyGsrSerDe/KafkaSerDe.py

```diff
@@ -0,0 +1,49 @@
+from .PyGsrSerDe import *
+from .AvroSerDe import *
+
+class GlueSchemaRegistryKafkaSerializer:
+    def __init__(self):  # data format design needs discussion
+        self.data_format = "AVRO"
+        self.gsr_serializer = GlueSchemaRegistrySerializer
+        serializers = {
```

Comment:
I think it might be a better idea if we initialize these data format serializer in a different way, one issue I see with this approach is, by the time we have all 3 data format serializer implemented, we will initialize all 3 of them regardless of which data format customer choose. That might cause performance issue

Reply:
Will fix in the protobuf pr - plan to change to factory